### DISCUSSION TITLE
两个PAG文件衔接播放的时候会存在跳空白帧的情况，求解

### DISCUSSION THREAD

**Discussion by gechaoyu** [Tencent/libpag#2138] [Q&A] [answered]
同样的工程之前是输出Lottie文件，可以衔接自然衔接播放，目前重新输出pag文件之后发现两个PAG文件衔接播放的时候会存在跳空白帧的情况，求解

**Comment by kevingpqi123** [comment, marked as answer]
如果需要衔接播放，建议通过 PAG 的组合模式，创建一个空的 PAGComposition 的，然后将两个 pag 文件添加进去，通过 setStartTime 接口可以设置第二个 pag 文件的真实播放时间，具体可以参考官方 demo 工程中的范例

**Reply by EleanWei** [reply]
业务场景对于第二个pag的需求是根据动态的音频时长来控制不确定次数的循环播放，这种情况怎么处理，在音频未播放结束之前是真实播放时长是不可知的。

**Comment by gechaoyu** [comment]
您好，您的邮件我已收到，我及时阅读~谢谢(づ￣ 3￣)づ

**Comment by gechaoyu** [comment]
您好，您的邮件我已收到，我及时阅读~谢谢(づ￣ 3￣)づ
